### PR TITLE
Added mechanical, propulsion and systems texts. Fix #7, fix #8, fix #9.

### LINKED ISSUE #7: Write text for mechanical team

### LINKED ISSUE #8: Write text for systems team

### LINKED ISSUE #9: Write text for propulsion team

### PR DESCRIPTION
**Pull request by stovatt**

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -167,8 +167,7 @@
               <div class="bg" style="background-image: url('/static/teams/mechanical.jpg')"></div>
               <header>Mechanical</header>
               <div class="text">
-                <div>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fugiat, veritatis, perferendis suscipit earum similique fuga quae quidem nesciunt tempore voluptate. Nihil minima ipsa, voluptates. Quos minus aspernatur in quis accusantium.</div>
-                <div>Quibusdam deserunt, nisi doloremque totam aperiam id saepe rem blanditiis, quaerat fugiat veritatis optio quidem nihil dolore atque. Facilis unde rem consequuntur earum veniam, id. Magni beatae nostrum, id consequuntur.</div>
+                <div>Mechanical team is responsible for designing and manufacturing our ground equipment. This includes a vertical test stand for a 3kN hybrid rocket engine and a launch rail. They are also the ones who manufacture the rocket body and integrate engine, electronics and parachute system into it. Their goal is to make sure every launch is a success.</div>
```
Comment (eweilow) [inline www/index.html]: I do not agree with the part about:
> Their goal is to make sure every launch is a success.

This should be the goal of every team.

Comment (stovatt) [reply www/index.html]: Ok, good point. Do you have any better suggestion? I tried to keep the texts as they were in the website texts document with only a few changes.

Comment (eweilow) [reply www/index.html]: Maybe we can just remove it?

Comment (stovatt) [reply www/index.html]: I would prefer to keep it to match the other teams.

**Pull request merged**